cmd/nerdctl/compose_ps_linux_test.go

```diff
@@ -95,6 +95,9 @@ volumes:
 	// check all services are up (can be marshalled and unmarshalled)
 	base.ComposeCmd("-f", comp.YAMLFullPath(), "ps", "--format", "json").
 		AssertOutWithFunc(assertHandler("all", 2, `"Service":"wordpress"`, `"Service":"db"`))
+	// check Image field
+	base.ComposeCmd("-f", comp.YAMLFullPath(), "ps", "--format", "json").
+		AssertOutWithFunc(assertHandler("all", 2, fmt.Sprintf(`"Image":"%s"`, testutil.WordpressImage), fmt.Sprintf(`"Image":"%s"`, testutil.MariaDBImage)))
```

Comment:
This can be merged into line 97 right? Since it accepts variable-length fields to be checked.

Reply:
Correct, changed it.